runners/katacoda/index.ts

```diff
@@ -338,6 +338,19 @@ export class Katacoda extends Runner {
         return null;
     }
 
+    runDockerCompose(step: Step, command: Command) : RunResult {
+        let terminal = this.getTerminal('runDockerCompose');
+        let cdCommand = this.changeCurrentDir(path.join(this.getVariable(this.workspaceDirectory), command.parameters[0]), terminal.terminalId, terminal.isRunning);
+
+        this.steps.push({
+            "title": "Execute Docker Compose",
+            "text": "step" +this.stepsCount + ".md"
+        });
+
+        this.renderTemplate("dockerCompose.md", this.outputPathTutorial + "step" + (this.stepsCount++) + ".md", { text: step.text, textAfter: step.textAfter, cdCommand: cdCommand, terminalId: terminal.terminalId, interrupt: terminal.isRunning, port: command.parameters[1].port, useDevonCommand: this.getVariable(this.useDevonCommand)});
```

Comment:
You do not need the useDevonCommand parameter as docker is not included in the ide